### PR TITLE
Add StyleCI and TravisCI setting files

### PR DESCRIPTION
**Pull request by morpheus7CS**
Hey @calebporzio (and @wotta),

I have opened a pull request, adding `.styleci`and `.travis.yml` files. This is the format that @mpociot also uses in his packages and recommends in his PHP Package Development course.

Two things:
1. @wotta is correct, the actual setup of those services needs to handled by @calebporzio.
2. Laravel uses StyleCI's multilanguage fixes, so the `laravel/framework`version of this file looks like this:

````yml
php:
  preset: laravel
js:
  finder:
    not-name:
      - webpack.mix.js
css: true

````

This is however not offered to open source repositories (yet) and requires their $10/month or $100/year plan.

Kind regards,
g

### CODE REVIEW

**Comment by loganhenson** [comment]
@morpheus7CS Thanks so much for the PR!

Unfortunately yea, the final setup of these services will likely need to wait for Caleb.

I do however think the TravisCI setup is fairly high immediate value here to cover the different framework / php versions, so I'd love to have a scoped PR for him when he gets back.

I'd like to specifically scope this PR down to:
- The TravisCI setup
- The test method typo
- the `.gitattributes` file
And remove the `.styleci` file for now.

I've merged a PR that fixes the hard coded system paths (after following your issue/commit for https://github.com/calebporzio/livewire/issues/76 🙌)

If you want to bother splitting the test method typo out into its own PR, we can even merge that right away.

Does that sound reasonable to you?

**Comment by morpheus7CS** [comment]
Hey @loganhenson,

nice job on fixing the test the right way. 

I have done merged your test in and removed .styleci, so this can be merged now.

Thanks,
g

File: src/Commands/LivewireDestroyCommand.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ public function handle()
 
         if (! $force = $this->option('force')) {
             $shouldContinue = $this->confirm(
-                "Are you sure you want to delete the following files?\n\n{$this->parser->classPath()}\n{$this->parser->viewPath()}\n"
+                "Are you sure you want to delete the following files?"
```
Comment (calebporzio) [inline src/Commands/LivewireDestroyCommand.php]: Why remove this from the output? I want to show the user what files they are about to delete

Comment (loganhenson) [reply src/Commands/LivewireDestroyCommand.php]: They were just trying to get the tests passing for CI. A PR was already merged that maintains the file output, and this PR should be caught up with that (https://github.com/calebporzio/livewire/pull/73/files/d057420a199eab70f524ef6a1c19d5b1525c3880..043abf410befb6f6babd0881119b781f38e35c54#diff-244259b80d8e7e333e1448a92aae04f0R22).

Comment (morpheus7CS) [reply src/Commands/LivewireDestroyCommand.php]: Hey @Caleb,

this was done just to keep the tests green for initial CI setup, but as @loganhenson has fixed the DestroyCommandTest in [#88](), we should keep command as it were. Missed it when resolving stuff in a hurry this morning.

Thanks,
g

**Comment by morpheus7CS** [comment]
Hey @calebporzio,

instead of you pruning out this, I’ll just close the PR and open another one just with the CI stuff. I can do that tonight, just let me know if you want the .styleci stuff in there or not.

Kind refards,
g

**Comment by morpheus7CS** [comment]
Will separate those out and close the current PR.

**Pull request closed without merging**